packages/@aws-cdk/custom-resource-handlers/lib/custom-resources/aws-custom-resource-handler/aws-sdk-v2-handler.ts

```diff
@@ -62,6 +62,16 @@ function patchSdk(awsSdk: any): any {
 
 /* eslint-disable @typescript-eslint/no-require-imports, import/no-extraneous-dependencies */
 export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent, context: AWSLambda.Context) {
+  event.ResourceProperties.Create = decodeCall(event.ResourceProperties.Create);
+  event.ResourceProperties.Update = decodeCall(event.ResourceProperties.Update);
+  event.ResourceProperties.Delete = decodeCall(event.ResourceProperties.Delete);
+
+  const call: AwsSdkCall | undefined = event.ResourceProperties[event.RequestType];
+
+  // if there is a call there will always be logging configured -- otherwise, in the event of no call, logging
+  // wasn't configure so just default to existing behavior
+  const logApiResponseData = call?.logApiResponseData ?? true;
```

Comment:
Do we need this in the v2 handler (I don't see where we use it)?

Reply:
A bit further down (lines 172 and 177). This is passed to `respond` which is in `shared.ts`. The v2 handler and the v3 handler share logic contained in `shared.ts`

Reply:
Got it! Lgtm!